app/constants/tasks-enum.js

```diff
@@ -1,20 +1,18 @@
 module.exports = {
   ACCEPT_CLAIM_NOTIFICATION: 'ACCEPT-CLAIM-NOTIFICATION',
+  ADVANCE_CLAIM_EVIDENCE_REMINDER_NOTIFICATION: 'ADVANCE-CLAIM-EVIDENCE-REMINDER-NOTIFICATION',
```

Comment:
This is just a re-order, makes it easier to check when it's alphabetically sorted.

Reply:
Some have been removed.